<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormResult</class>
 <widget class="QDialog" name="FormResult">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>410</y>
     <width>331</width>
     <height>29</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>11</y>
     <width>441</width>
     <height>391</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Создан</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QDateTimeEdit" name="ed_create">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Номер</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="lb_person">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Спортсмен</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Старт</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Финиш</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Отсечка</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Штраф</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Результат</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Статус</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>Комментарий</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QTimeEdit" name="ed_start">
      <property name="displayFormat">
       <string>HH:mm:ss.z</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QTimeEdit" name="ed_finish">
      <property name="displayFormat">
       <string>HH:mm:ss.z</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QTimeEdit" name="ed_result">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="displayFormat">
       <string>HH:mm:ss.z</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QTimeEdit" name="ed_offset">
      <property name="displayFormat">
       <string>HH:mm:ss.z</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QTimeEdit" name="ed_penalty">
      <property name="displayFormat">
       <string>HH:mm:ss.z</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QSpinBox" name="ed_bib">
      <property name="maximum">
       <number>99999999</number>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QComboBox" name="cb_status">
      <item>
       <property name="text">
        <string>-</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ОК</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Финишировал</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ДИСКВ.</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Непр. отметка</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Сошёл</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Активный</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Неактивный</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ПРЕВЫШЕНО КВ</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Проведена жеребьёвка</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Вне конкурса</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Перемещён</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Перемещён наверх</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>НЕ СТАРТ.</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Не заявлен</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Заявка отменена</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Восстановлен</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="ed_comment"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FormResult</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FormResult</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
